tests/func-tests/node_placement_test.go

```diff
@@ -146,6 +147,10 @@ var _ = Describe("[rfe_id:4356][crit:medium][vendor:cnv-qe@redhat.com][level:sys
 				delete(expectedWorkloadsPods, "secondary-dns")
 			}
 
+			if cnaoCR.Spec.KubevirtIpamController == nil {
```

Comment:
I see that it follows the pattern above, but to me it makes more sense to add the value to the map, if cnaoCR.Spec.KubevirtIpamController != nil, than this pattern.

Reply:
updated